diff --git a/config.sh b/config.sh
index 7653731..a6b2ea8 100755
--- a/config.sh
+++ b/config.sh
@@ -687,7 +687,7 @@ grep CS_VERSION globals.h | cut -d\" -f2
 		break
 	;;
 	'-r'|'--oscam-revision')
-		revision=`(svnversion -n . 2>/dev/null || printf 0) | sed 's/.*://; s/[^0-9]*$//; s/^$/0/'`
+		echo "$OSCAM_ADDON_VERSION"
 		if [ "$revision" = "0" ] 
		then
			which git > /dev/null 2>&1 && revision=`git log -1 --pretty=%B | grep git-svn-id |
