--- a/src/soxr.pc.in	2016-03-14 14:31:51.000000000 +0100
+++ b/src/soxr.pc.in	2018-04-10 17:07:00.513809916 +0200
@@ -1,5 +1,5 @@
 Name: ${PROJECT_NAME}
 Description: ${DESCRIPTION_SUMMARY}
-Version: ${PROJECT_VERSION}
+Version: 0.1.3
 Libs: -L${LIB_INSTALL_DIR} -l${PROJECT_NAME}
 Cflags: -I${INCLUDE_INSTALL_DIR}

 --- a/src/soxr-lsr.pc.in	2016-03-14 14:31:51.000000000 +0100
 +++ b/src/soxr-lsr.pc.in	2018-04-10 17:12:00.012155060 +0200
 @@ -1,5 +1,5 @@
  Name: ${LSR}
  Description: ${DESCRIPTION_SUMMARY} (with libsamplerate-like bindings)
 -Version: ${PROJECT_VERSION}
 +Version: 0.1.3
  Libs: -L${LIB_INSTALL_DIR} -l${LSR}
  Cflags: -I${INCLUDE_INSTALL_DIR}
