--- a/xbmc/guilib/GUIWindowManager.cpp	2018-04-15 20:19:40.729676337 +0200
+++ b/xbmc/guilib/GUIWindowManager.cpp	2018-04-15 20:25:33.870181722 +0200
@@ -848,7 +848,16 @@
   int currentWindow = GetActiveWindow();
   CGUIWindow *pWindow = GetWindow(currentWindow);
   if (pWindow)
-    CloseWindowSync(pWindow, iWindowID);
+  {
+    if (iWindowID == WINDOW_SCREENSAVER)
+    {
+      pWindow->Close(true, iWindowID);
+    }
+    else
+    {
+      CloseWindowSync(pWindow, iWindowID);
+    }
+  }
   CServiceBroker::GetGUI()->GetInfoManager().GetInfoProviders().GetGUIControlsInfoProvider().SetNextWindow(WINDOW_INVALID);

   // Add window to the history list (we must do this before we activate it,
