--- a/xbmc/windowing/amlogic/WinSystemAmlogic.cpp  2018-04-16 13:38:58.907441295 +0200
+++ b/xbmc/windowing/amlogic/WinSystemAmlogic.cpp  2018-04-16 15:49:55.244827848 +0200
@@ -104,6 +104,9 @@

   aml_set_framebuffer_resolution(1920, 1080, m_framebuffer_name);

+  m_lirc.reset(OPTIONALS::LircRegister());
+  CLog::Log(LOGDEBUG, "OPTIONALS::LircRegister -- register Lirc client");
+
   return CWinSystemBase::InitWindowSystem();
 }
 
 --- a/xbmc/windowing/amlogic/WinSystemAmlogic.h  2018-04-16 10:06:21.000000000 +0200
 +++ b/xbmc/windowing/amlogic/WinSystemAmlogic.h  2018-04-16 15:49:42.296431490 +0200
 @@ -21,6 +21,7 @@
  #pragma once

 +#include "platform/linux/OptionalsReg.h"
  #include "rendering/gles/RenderSystemGLES.h"
  #include "threads/CriticalSection.h"
  #include "windowing/WinSystem.h"
  #include "threads/SystemClock.h"
 @@ -63,4 +64,5 @@

    CCriticalSection m_resourceSection;
    std::vector<IDispResource*> m_resources;
 +  std::unique_ptr<OPTIONALS::CLircContainer, OPTIONALS::delete_CLircContainer> m_lirc;
  };
